### PR TITLE
prepare_sector: add biomass transport costs for networks wo biomass network

### PR DESCRIPTION
**Pull request by FabianHofmann**
Add biomass transport cost to nodes in case no biomass transport network is considered.

### CODE REVIEW

File: rules/build_sector.smk [hunk 1]
```diff
@@ -782,7 +772,7 @@ rule prepare_sector_network:
         + "prenetworks/elec_s{simpl}_{clusters}_l{ll}_{opts}_{sector_opts}_{planning_horizons}.nc",
     threads: 1
     resources:
-        mem_mb=2000,
+        mem_mb=20000,
```
Comment (lisazeyen) [inline rules/build_sector.smk:775]: that's quite a large increase, is that much memory necessary?

File: scripts/prepare_sector_network.py [hunk 2]
```diff
@@ -2185,6 +2185,23 @@ def add_biomass(n, costs):
             carrier="solid biomass transport",
         )
 
+    elif options["biomass_spatial"]:
```
Comment (lisazeyen) [inline scripts/prepare_sector_network.py:2188]: What if biomass is not spatially resolved (just one EU node for the biomass). Should the transport costs be added? If not one could make the rule to build the biomass transport costs conditional again. If yes, the generator should be added here as well.

**Review by lisazeyen — CHANGES_REQUESTED** [review]

File: rules/build_sector.smk (hunk 1 above)
Comment (FabianHofmann) [reply rules/build_sector.smk:775]: that was actually not intended, thanks for pointing out

File: scripts/prepare_sector_network.py (hunk 2 above)
Comment (FabianHofmann) [reply scripts/prepare_sector_network.py:2188]: I see, haven't thought about this option. Will make it conditional again

**Comment by FabianHofmann** [comment]
thanks @lisazeyen, I fixed the things, please approve if you are fine with the changes

**Review by lisazeyen — APPROVED** [review]
Great, looks good!

**Pull request merged**